server/src/session.rs

```diff
@@ -42,38 +43,39 @@ impl Session {
         // Use the root_uri if it exists temporarily as we cannot access configuration until
         // after initialization. Additionally, LSP may provide the windows path with escaping or a lowercase
         // drive letter. To fix this, we convert the path to a URL and then back to a path.
-        let root_uri = params
+        let root_path = params
             .root_uri
             .and_then(|uri| uri.to_file_path().ok())
             .and_then(|path| Url::from_file_path(path).ok())
+            .and_then(|uri| uri.to_file_path().ok())
             .expect("root_uri not found in initialization parameters");
```

Comment:
Should this expect be updated? I don't ever expect it to be emitted but its a bit better than just unwrapping.

Reply:
Sure, I updated the expect to be more correct.